ArduPlane/Parameters.h

```diff
@@ -437,7 +441,6 @@ class Parameters {
     AP_Int16 dspoiler_rud_rate;
     AP_Int32 log_bitmask;
     AP_Int32 RTL_altitude_cm;
-    AP_Int16 pitch_trim_cd;
```

Comment:
it should just change to AP_Float

Reply:
It had already been partly moved to AP_FixedWing;:aparm .. I'm just completing the process. See the above discussion about #22191

Reply:
it doesn't need to be moved

Reply:
fixed